yaml/pois.yaml

```diff
@@ -389,7 +394,52 @@ filters:
     min_zoom: 17
     output: {kind: office}
   - filter:
-      shop: [bakery, bicycle, books, butcher, car, car_repair, clothes, computer, convenience,
+      shop: [bicycle]
```

Comment:
Many bicycle shops sometimes also rent bicycles, that's buried in some service tags. That's beyond the scope of this issue.